packages/itwin/ec3-widget/src/components/EC3/EC3Config.ts

```diff
@@ -96,11 +96,27 @@ export type EC3ConfigPropsWithDefaultReport = EC3ConfigCommonProps & {
   defaultReport?: Report;
 };
 
+export type EC3ConfigPropsWithCallbacks = EC3ConfigCommonProps & {
+  /**
+   * The callback function for EC3 export success.
+   */
+  onExportSucceeded?: (status: EC3JobStatus) => void;
+
+  /**
+   * The callback function for EC3 export failure.
+   */
+  onExportFailed?: (status: EC3JobStatus) => void;
+};
```

Comment:
This looks like it can be combined and be more generic. e.g. `onExportResult?: (status:EC3JobStatus) => void`